### ISSUE TITLE
HW5 ready for grading!

### ISSUE THREAD

**Issue by gabriellindqvist** [gabriellindqvist/su-mt4007#5]

**Comment by Chromatina** [comment]
- General
Yes the person has answered all the questions. I really like that you explain what you're going to do before you do it and that you write functions and variables using a `code` block since it makes it easier for the reader to follow along. 
- REST
It seems like you forgot to load the library `RColorBrewer` as there is an error `Loading required package: RColorBrewer` under that code block. Although in the code you use `suppressPackageStartupMessages(library(RColorBrewer))` which seems to have worked in the previous code blocks with for example the package `tm`. Furthermore, there isn't a lot of words in the word cloud even though you use `max.words = 100` when displaying the word cloud, which seems odd since there definitely is more than just 20 unique words in all of the prize motivations. 
- Web Scraping
In this task you must have forgotten to end the code blocks since all of your code and text is in the same code block. This is probably also the reason to all the white space in this code block. The dataframe containing the 60 books seems to display the correct information although the dataframe itself is not being displayed correctly.

**Comment by gabriellindqvist** [comment]
> Furthermore, there isn't a lot of words in the word cloud even though you use `max.words = 100` when displaying the word cloud, which seems odd since there definitely is more than just 20 unique words in all of the prize motivations.

I agree that there definitely more than just 20 unique words in **all** of the prize motivations. My understanding was to fetch only prize motivations relevant to the physics category.

>   In this task you must have forgotten to end the code blocks since all of your code and text is in the same code block. This is probably also the reason to all the white space in this code block. The dataframe containing the 60 books seems to display the correct information although the dataframe itself is not being displayed correctly.

For some reason it displays differently on here on GitHub (just like you described). On jupyter lab it displays everything correctly.

Thank you for your feedback :)

**Comment by LeoLevenius** [comment]
# $${\color{green}\text{Godkänt.}}$$
